src/appkit.cs

```diff
@@ -16733,7 +17163,7 @@ public interface NSTextStorageDelegate {
 	}
 
 	[BaseType (typeof (NSObject))]
-	public interface NSTextTab : NSCoding, NSCopying {
+	public interface NSTextTab : NSSecureCoding, NSCopying {
```

Comment:
@spouliot isn't this a breaking change? Not sure if we want to keep this backwards compat here